soc/nxp/imx/imx8m/Kconfig.defconfig.imx8mp_adsp

```diff
@@ -0,0 +1,55 @@
+# Copyright 2024 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+if SOC_SERIES_IMX_8M
+
+if SOC_MIMX8MP_ADSP
+
+config SOC
+	default "mimx8ml8"
+
+config SOC_TOOLCHAIN_NAME
+	string
+	default "nxp_imx8m_adsp"
```

Comment:
`Kconfig.soc`

Reply:
This SOC_TOOLCHAIN_NAME will not apply for all cores from SoC imx8m. This is only for DSP, which uses xtensa: i.e. /opt/zephyr-sdk-0.16.4/xtensa-nxp_imx8m_adsp_zephyr-elf/